reference/core/ingress-controller.md

```diff
@@ -171,7 +171,7 @@ will set up the Ambassador Edge Stack to do canary routing where 50% of the traf
 An `Ingress` resource must provide at least some routes or a [default backend](https://kubernetes.io/docs/concepts/services-networking/ingress/#default-backend). The default backend provides for a simple way to direct all traffic to some upstream service:
 
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.